code-challenges.test.js

```diff
@@ -64,4 +152,25 @@ const vowelTester3 = "challenges"
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+const firstVowels = (string) => {
+    for (let i = 0; i < string.length; i++) {
+        let vowel = string.charAt(i)
+        if (vowel === "a" || vowel === "e" || vowel === "i" || vowel === "o" || vowel === "u") {
```

Comment:
Instead of the vowel variable, you can just use string[i] to access the letter at each iteration. This approach is just fine, but since you were wondering about a refactor, there are built-in methods available to you that you can use for this challenge. Look up `.search` and see how you can implement it for this challenge 🙂